db/schema.rb

```diff
@@ -23,6 +23,12 @@
     t.datetime "verified"
   end
 
+  create_table "ar_internal_metadata", primary_key: "key", force: :cascade do |t|
```

Comment:
This is Rails 5 stuff. I'm guessing it leaked into master because of an issue in my changing branches while developing.  I don't think its doing any harm, so I'll leave it for now.